assets/src/components/lateView.tsx

```diff
@@ -36,8 +58,187 @@ const compareGhosts = (a: Ghost, b: Ghost): number => {
   return runIdToLabel(a.runId!).localeCompare(runIdToLabel(b.runId!))
 }
 
+const idIn = (runId: RunId, runIds: RunId[]): boolean =>
+  runIds.some((idInList) => idInList === runId)
```

Comment:
*suggestion (non-blocking):* `runIdIn` may be a better name if it's specific to that type.